docs/current_docs/cookbook/snippets/cache-dependencies/php/src/MyModule.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Container;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class MyModule
+{
+    // Build an application using cached dependencies
+    #[DaggerFunction]
+    public function build(
+        // source code location
+        Directory $source,
+    ): Container
+    {
```

Comment:
```suggestion
    ): Container {
```

This one is technically a coding standard: https://www.php-fig.org/per/coding-style/#45-method-and-function-parameters